javascript/javascript-playground-questions/README.md

```diff
@@ -35,3 +35,5 @@ sections:
     - js-beginning-and-end-pairs
     - js-harmonic-series
     
+next:
+  - javascript:react

```

Comment:
we can remove this `next` here, the js topic doesn't have a react course anymore (it used to have one in the past)
```suggestion
```